NomadAudio/src/SampleRateConverter.cpp

```
@@ -1,13 +1,57 @@
+#include <mutex>
+#include <map>
 
 namespace Nomad {
 namespace Audio {
 
+// =============================================================================
+// Shared Filter Bank Cache
+// =============================================================================
+
+namespace {
+    struct FilterCache {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: mutex

(cppcoreguidelines-pro-type-member-init)